datalad/interface/clean.py

```diff
@@ -91,13 +94,17 @@ def __call__(dataset=None, what=None, recursive=False, recursion_limit=None):
                 continue
             d = ap['path']
             gitdir = GitRepo.get_git_dir(d)
+            DIRS_PLURAL = ("directory", "directories")
+            FILES_PLURAL = ("file", "files")
             for dirpath, flag, msg, sing_pl in [
                 (ARCHIVES_TEMP_DIR, "cached-archives",
-                 "temporary archive", ("directory", "directories")),
+                 "temporary archive", DIRS_PLURAL),
                 (ANNEX_TEMP_DIR, "annex-tmp",
-                 "temporary annex", ("file", "files")),
+                 "temporary annex", FILES_PLURAL),
+                (ANNEX_TRANSFER_DIR, "annex-transfer",
```

Comment:
Shouldn't you add "annex-transfer" to the `what` option (or perhaps just classify this as "annex-tmp")?

Reply:
ho ho -- thank you @kyleam for the thorough review!  This gotcha shows again that duplication is evil, will RF a bit to avoid it! ;)